lib/maildir.rb

```diff
@@ -35,6 +35,7 @@ def initialize(path, create = true)
     @path = File.expand_path(path)
     @path = File.join(@path, '/') # Ensure path has a trailing slash
     @path_regexp = /^#{Regexp.quote(@path)}/ # For parsing directory listings
+    @serializer = nil
```

Comment:
There's no associated test for this change. What problem is this solving? It seems like:

    def serializer
      @serializer || @@serializer
    end

is sufficient unless there's a case where @serializer is truthy but not a @@serializer, in which case assigning nil is insufficient anyway.